codegenerator/cli/npm/envio/src/Utils.res

```diff
@@ -205,6 +225,9 @@ Helper to check if a value exists in an array
     })
     interleaved
   }
+
+  @send
+  external flatten: (array<array<'a>>, @as(json`1`) _) => array<'a> = "flat"
```

Comment:
Interesting, never done a binding like this with hardcoded params :) does @as(1) not work as well out of interest?

Reply:
Yeah, it works 😁 I didn't know that it does :)